jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletMultiPartFormData.java

```diff
@@ -83,7 +83,7 @@ static Parts getParts(ServletRequest servletRequest)
     static void onParts(ServletRequest servletRequest, String contentType, Promise.Invocable<Parts> promise)
     {
         CompletableFuture<Parts> futureParts = from(servletRequest, promise.getInvocationType(), contentType);
-        futureParts.whenComplete(promise);
+        futureParts.whenComplete(Promise.Invocable.toBiConsumer(promise));
```

Comment:
We need to open an issue to get rid of the implementations using the deprecated CompletableFuture APIs